.gitignore

```diff
@@ -141,3 +141,12 @@ doc/source/savefig/
 # Pyodide/WASM related files #
 ##############################
 /.pyodide-xbuildenv-*
+
+
+# Web & Translations #
+##############################
+web/preview/
+web/pandas-translations-main/
+web/pandas/es/
+web/pandas/pt/
+web/pandas/fr/
```

Comment:
Not sure if the two first items are relevant anymore. Also, since the languages will keep changing, not sure if I'd delete all these ignores, so we don't need to maintain them here too. But up to you.